src/components/App.js

```diff
@@ -1,14 +1,81 @@
 import React from 'react';
+import Search from './Search';
+import Results from './Results';
+import Hero from './Hero';
 
 class App extends React.Component {
   constructor(){
     super();
+
+    this.handleClick = this.handleClick.bind(this)
+    this.handlePrevClick = this.handlePrevClick.bind(this)
+    this.receiveSearch = this.receiveSearch.bind(this)
+   
+    this.fetchMovies = this.fetchMovies.bind(this)
+    this.state ={
+        userSearch: "",
+        moviesArray:[],
+        page: 1
+    }
+  }
+
+  
+  fetchMovies(userSearch){
+    fetch(`http://www.omdbapi.com/?apikey=8d5ab09&s=${userSearch}&page=${this.state.page}`)
+    .then(response => response.json())
+    .then(body => {
+      // console.log(body)
+      this.setState({moviesArray: body.Search,
+                    
+      })
+    })
+  }
+
+
+// function receive user input
+  receiveSearch(text){
+    this.setState({
+      userSearch: text
+    })
+    this.fetchMovies(text)
   }
 
+
+  handleClick(event){
+   
+    this.setState({
+      page: this.state.page + 1
+    }, () => this.fetchMovies(this.state.userSearch,this.setState.page))
```

Comment:
to access page from state you need to use `this.state.page` rather than `this.setState.page`. `setState` is a method used to change state rather than get values